mlflow/tracking/fluent.py

```diff
@@ -212,6 +212,40 @@ def log_metric(key, value):
     MlflowClient().log_metric(run_id, key, value, int(time.time()))
 
 
+def log_metrics(metrics):
+    """
+    Log multiple metrics for the current run, starting a run if no runs are active.
+    :param metrics: Dictionary of metric_name: String -> value: Float
+    :returns: None
+    """
+    run_id = _get_or_start_run().info.run_uuid
+    timestamp = int(time.time())
+    metrics_arr = [Metric(key, value, timestamp) for key, value in metrics.items()]
+    MlflowClient().log_batch(run_id=run_id, metrics=metrics_arr, params=[], tags=[])
+
+
+def log_params(params):
+    """
+    Log a batch of params for the current run, starting a run if no runs are active.
+    :param params: Dictionary of param_name: String -> value: String
```

Comment:
We should mention that non-string values will be stringified, like we do in the documentation for `log_param()`.